src/persistence/historykeeper.cpp

```diff
@@ -146,6 +167,21 @@ HistoryKeeper::HistoryKeeper(GenericDdInterface *db_) :
         messageID = sqlAnswer.value(0).toLongLong();
 }
 
+void HistoryKeeper::importAvatar(const QString& ownerId)
```

Comment:
could you rename this method to `importAvatarToDatabase`, just `importAvatart` seems to be misleading for me.